### PR TITLE
Add autoselect optional parameter to select_produits

### PR DESCRIPTION
**Pull request by borisduin**
# New Add optional Autoselect parameter for Select_produits

Add optional Autoselect parameter for Select_produits for used in ajax product select call.

### CODE REVIEW

File: htdocs/core/class/html.form.class.php [hunk 1]
```diff
@@ -1891,9 +1891,10 @@ public function select_dolusers_forevent($action = '', $htmlname = 'userid', $sh
 	 *										            'warehouseclosed' = select products from closed warehouses,
 	 *										            'warehouseinternal' = select products from warehouses for internal correct/transfer only
 	 *  @param 		array 		$selected_combinations 	Selected combinations. Format: array([attrid] => attrval, [...])
+	 *  @param 		int 		$autoselect 			Autoselect option for Ajax Product select
```
Comment (eldy) [inline htdocs/core/class/html.form.class.php]: Can you confirm this feature means "Automatic selection if just one value" ?

File: htdocs/product/ajax/products.php [hunk 2]
```diff
@@ -187,9 +188,9 @@
 
 	$form = new Form($db);
 	if (empty($mode) || $mode == 1) {  // mode=1: customer
-		$arrayresult = $form->select_produits_list("", $htmlname, $type, 0, $price_level, $searchkey, $status, $finished, $outjson, $socid, '1', 0, '', $hidepriceinlabel, $warehouseStatus);
+		$arrayresult = $form->select_produits_list("", $htmlname, $type, $limit, $price_level, $searchkey, $status, $finished, $outjson, $socid, '1', 0, '', $hidepriceinlabel, $warehouseStatus);
```
Comment (eldy) [inline htdocs/product/ajax/products.php]: It seems the PR contains change from another PR.

Comment (borisduin) [reply htdocs/product/ajax/products.php]: Umm. I'll correct that

**Comment by eldy** [comment]
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.

**Pull request closed without merging**